install/update_0905_91.php

```diff
@@ -174,11 +174,19 @@ function update0905to91() {
       foreach ($CFG_GLPI['lock_lockable_objects'] as $itemtype) {
          $rightnames[] = "'".$itemtype::$rightname."'";
       }
-      $query = "UPDATE `glpi_profilerights`
-                SET `rights` = `rights` | ".UNLOCK."
-                WHERE `profiles_id` = '4'
-                      AND `name` IN (".implode( ",", $rightnames ).")";
-      $DB->queryOrDie($query, "update super-admin profile with UNLOCK right");
+
+      $DB->updateOrDie("glpi_profilerights", [
+            'rights' => new \QueryExpression(
+               DBmysql::quoteName("rights") . " | " . DBmysql::quoteValue(UNLOCK)
+            )
+         ], [
+            'profiles_id'  => 4,
+            new \QueryExpression(
```

Comment:
You can use `'name' => $rightnames`. It will use the IN operator as value is an array.
You can also remove the quotes on line 175, will be added automatically.